cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -2969,15 +2969,16 @@ def _create_temp_backing_from_attached_vmdk(
                   "%(instance)s.", {'dev': vol_dev_uuid,
                                     'instance': instance})
 
-        tmp_name = tmp_name or uuidutils.generate_uuid()
+        tmp_name = tmp_name or "TEMP_BACKING-%s" % uuidutils.generate_uuid()
 
         device_changes = self.volumeops._create_device_change_for_disk_removal(
             instance, disks_to_clone=[vol_dev_uuid])
         device_changes.extend(
             self.volumeops._create_device_change_for_vif_removal(instance))
 
         # Remove another attribute by which the nova driver identifies VMs
-        extra_config = {'nvp.vm-uuid': ''}
+        extra_config = {'nvp.vm-uuid': '', 'hardware.numCPU': 1,
+                        'hardware.memoryMB': 128}
```

Comment:
Sorry, `hardware.numCPU` and `hardware.memoryMB` are not `extraConfig` keys. These are actual keys on the `VirtualMachine` object and need to be reconfigured differently.

I think changing them should be a follow-up if we want to do it at all.